src/AdapterPluginManager.php

```diff
@@ -45,7 +58,7 @@ class AdapterPluginManager extends AbstractPluginManager
      * @return void
      * @throws Exception\RuntimeException if invalid
      */
-    public function validatePlugin($plugin)
+    public function validate($plugin)
```

Comment:
That's okay, and intended; it ensures that the exceptions thrown from plugin managers are consistent.